pkg/pipeline/validation/task_validations.go

```diff
@@ -1,58 +1,102 @@
 package validation
 
 import (
-	"fmt"
+	errors2 "errors"
```

Comment:
Please remove unnecessary alias `error2`

Reply:
IT is needed, but I suggest aliasing it to stderrors